src/main/resources/archetype-resources/pom.xml

```diff
@@ -130,13 +130,13 @@
 		<repository>
 			<id>cumulocity</id>
 			<layout>default</layout>
-			<url>http://download.cumulocity.com/maven/repository</url>
+			<url>https://download.cumulocity.com/maven/repository</url>
```

Comment:
I understood that the cumulocity maven repository could be removed?

Reply:
unfortunately they are still required.